target/linux/ramips/dts/mt7621_linksys_ea7500-v2.dts

```diff
@@ -0,0 +1,206 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "linksys,ea7500-v2", "mediatek,mt7621-soc";
+	model = "Linksys EA7500 v2";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wan_green {
+			label = "ea7500-v2:green:wan";
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+		};
+
+		lan4_green {
```

Comment:
Why is the order on names inverted compared to the labels? Only one can be right ...